### PR TITLE
Adds filtering to Sinopia search

### PR DESCRIPTION
**Pull request by justinlittman**

### CODE REVIEW

**Comment by justinlittman** [comment]
image

**Comment by justinlittman** [comment]
Just a note that I'm going to do class filterating separate from group filtering.

**Comment by justinlittman** [comment]
image

File: src/components/search/TypeFilter.jsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  const clearFilter = () => {
+    dispatch(fetchSinopiaSearchResults(query, { ...searchOptions, startOfRange: 0, typeFilter: undefined }))
+    setTypeFilterShowDropdown(false)
+  }
+
+  const handleOnly = (type) => {
+    setSelectedTypeFilters([type])
+  }
+
+  const [typeFilterShowDropdown, setTypeFilterShowDropdown] = useState(false)
```
Comment (jermnelson) [inline src/components/search/TypeFilter.jsx]: Not a blocker but should these state functions be moved to line 11 to make it more consistent with other components?

**Pull request merged**